platform/app/src/routes/Mode/Mode.tsx

```diff
@@ -43,6 +45,46 @@ function defaultRouteInit(
         SeriesInstanceUID
       );
 
+      /**
+       * This function is used to check if the filter is used. Its intend is to
+       * warn the user in case of link with a SeriesInstanceUID was called
+       * @param instances
+       * @returns
+       */
+      function isSeriesFilterUsed(instances) {
+        const seriesInstanceUIDs = filters?.seriesInstanceUID;
+        if (seriesInstanceUIDs) {
+          if (instances.length) {
+            const instance = instances[0];
+            if (seriesInstanceUIDs.includes(instance.SeriesInstanceUID)) {
+              return true;
+            } else {
+              return false;
+            }
+          } else {
+            return false; // no images so the filter didn't work
+          }
+        } else {
+          return true; // no filter so its ok
+        }
```

Comment:
this can written better

```
if (!series){
  return true
}

if (!instances.length){
  return false
}

return seriesInstanceUIDs.includes(instances[0].SeriesInstanceUID);
```